news/_posts/core-weekly/2022-10-17-coreweekly-41-2022.md

```diff
@@ -0,0 +1,193 @@
+* [#105](https://github.com/PrestaShop/example-modules/pull/105): Demodoctrine: nullable Quote's content for nonDefault language. Thank you [@leemyongpakvn](https://github.com/leemyongpakvn)
+* [#100](https://github.com/PrestaShop/example-modules/pull/100): DemoGrid - add new columns and filters, add support for PS 1.7.7. Thank you [@leemyongpakvn](https://github.com/leemyongpakvn)
+
+
+### Changes in developer documentation sources
+* [#1555](https://github.com/PrestaShop/docs/pull/1555): Re-order items, fix minor typo in core changes for 8.x. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1553](https://github.com/PrestaShop/docs/pull/1553): Module extension : webservices - Backport 1410. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1552](https://github.com/PrestaShop/docs/pull/1552): Fix KpiRowHook location - backport 1471. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1551](https://github.com/PrestaShop/docs/pull/1551): Add doc about sendemail on webservices / order_carriers - Backport 1475. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1550](https://github.com/PrestaShop/docs/pull/1550): Update webservice reference.md, fix stock_availables methods - Backport 1480. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1549](https://github.com/PrestaShop/docs/pull/1549): Change referenced module name - backport 1506. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1548](https://github.com/PrestaShop/docs/pull/1548): Update instructions on how to fill PR. Backport of #1528. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1547](https://github.com/PrestaShop/docs/pull/1547): Install module from cli and clean doc - Backport of 1534. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1546](https://github.com/PrestaShop/docs/pull/1546): Install from cli documentation - Backport of 1538. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1542](https://github.com/PrestaShop/docs/pull/1542): Change urls from build.prestashop.com to build.prestashop-project.org for 1.7.x. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1541](https://github.com/PrestaShop/docs/pull/1541): Change urls from build.prestashop.com to build.prestashop-project.org for 8.x. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1538](https://github.com/PrestaShop/docs/pull/1538): Add install method from cli in doc. Thank you [@thomasnares](https://github.com/thomasnares)
+* [#1536](https://github.com/PrestaShop/docs/pull/1536): Fix release lifecycle links (1.7.x backport), by [@matks](https://github.com/matks)
+* [#1534](https://github.com/PrestaShop/docs/pull/1534): Add installation method module and remove duplicated doc from configuration. Thank you [@thomasnares](https://github.com/thomasnares)
+
+
+### ps-org-theme
```

Comment:
To be replaced by a more user-oriented name

Reply:
https://github.com/PrestaShop/core-weekly-generator/pull/128